spec/serializers/manifest_serializer_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+describe ManifestSerializer do
```

Comment:
nice! I am never sure how or when to unit test serializers, but this seems like a great use case